### PR TITLE
update stats sidebar & add tidelift dev practices

### PR DESCRIPTION
**Pull request by bee64**
update the stats sidebar styling and add a "Development practices" section

https://github.com/librariesio/libraries.io/assets/7821335/314e17a8-a7ad-4a9b-b34f-19be3c85a1be

### CODE REVIEW

**Comment by bee64** [comment]
realized this bumps the Documentation down, fixin that up real quick

File: app/views/projects/_dev_practices.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  <div class='col-xs-12 has-margin-top'>
+    <div class='has-margin-top'>
+      <%= link_to image_tag('Tidelift-logo-on-light.svg', width: 160), 'https://tidelift.com/?utm_source=librariesio&utm_medium=referral&utm_campaign=dev-practices' %>
+    </div>
+    <p class='tidelift-text has-margin-top'>
+      The Tidelift Subscription provides access to a continuously curated stream of <b>human-researched and maintainer-verified data</b> on open source packages and their licenses, releases, vulnerabilities, and development practices.
+    </p>
+    <a
+      type='button'
+      href='https://tidelift.com/?utm_source=librariesio&utm_medium=referral&utm_campaign=dev-practices'
```
Comment (wenottingham) [inline app/views/projects/_dev_practices.erb:38]: Can this go to https://tidelift.com/solutions/open-source-data ?

Comment (bee64) [reply app/views/projects/_dev_practices.erb:38]: yep!

**Comment by bee64** [comment]
updated the bootstrap grid layout to use two columns instead of two rows 

https://github.com/librariesio/libraries.io/assets/7821335/f55c2e3f-335c-4dcc-890a-5c84354d557e

**Review by kjmahalingam — APPROVED** [review]

**Review by djpowers — APPROVED** [review]

**Pull request merged**